portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -2485,7 +2485,127 @@ definition {
 
 		Form.viewFormDescription(formDescription = "Form Description");
 
-		FormFields.viewTextLabel(fieldLabel = "Text");
+		FormFields.viewTextLabel(fieldLabel = "Age");
 	}
 
+	@description = "Test for creating a form, with changes in the language checking the content sent"
+	@priority = 5
+	test NewTestFormWithMacro {
+
+		task ("Creating a Form with Description") {
```

Comment:
Look for your task descriptions, they are not in Liferay Default. Please, make tasks using BDD Framework. Review and send it again.